### PR TITLE
Pass config_path to uploader

### PR DESCRIPTION
**Pull request by equinor-ruaj**
Tests will fail until https://github.com/equinor/fmu-sumo-uploader/pull/89 is merged.

### CODE REVIEW

File: src/fmu/sumo/sim2sumo/common.py [hunk 1]
```diff
@@ -385,7 +392,12 @@ def add(self, file):
 
     def _upload(self):
         self._logger.debug("%s files to upload", len(self._objects))
-        nodisk_upload(self._objects, self._parentid, connection=self._conn)
+        nodisk_upload(
```
Comment (perolavsvendsen) [inline src/fmu/sumo/sim2sumo/common.py:395]: Is this only used internally in sim2sumo?

**Comment by perolavsvendsen** [comment]
I don't understand how the `config_path` argument actually makes it from the `SIM2SUMO` forward model job and to the uploader. I see it coming in as an argument, i.e. to `args.config_path` but I cannot see it going from there into the call to fmu-sumo-uploader 🤷‍♂️

**Comment by equinor-ruaj** [comment]
> I don't understand how the `config_path` argument actually makes it from the `SIM2SUMO` forward model job and to the uploader. I see it coming in as an argument, i.e. to `args.config_path` but I cannot see it going from there into the call to fmu-sumo-uploader 🤷‍♂️

The change in `main.py` passes the config_path to a `Dispatcher(one_datafile, args.env, config_path=args.config_path)`
This is stored as `self._config_path` which is used in `_upload()`
`_upload()` is used internally by sim2sumo (the `Dispatcher`) either when enough files have been `add()`ed or when `finish()` is called.

**Review by perolavsvendsen — APPROVED** [review]
LGTM 🤷‍♂️

File: src/fmu/sumo/sim2sumo/common.py (hunk 1 above)
Comment (equinor-ruaj) [reply src/fmu/sumo/sim2sumo/common.py:395]: Kind off 🤷

**Pull request merged**